README.rst

```diff
@@ -100,3 +100,39 @@ API
   **default** - The value to return if the environment variable is not present.
   (providing a default alongside setting ``required=True`` will raise a
   ``ValueError``)
+
+* ``excavator.env_timestamp(name, required=False, default='', format='timestamp')``::
```

Comment:
Not sure what you mean. The README.rst doc seems to be out of sync with the code. In particular the `format` parameter no longer exists.